ckanext/api_tracking/templates/dashboard/dataset-unique-views.html

```diff
@@ -5,13 +5,13 @@
     
     <section id="stats-dataset-views" class="module-content tab-content active">
       <h2>{{ _('Most Viewed Datasets') }}</h2>
-      <p>Unique visits</p>
+      <p>{{ _("Unique visits") }}</p>
       <p>
-        This dashboard consider "Unique visits" as the number of unique users that have visited a dataset
-        in the last year, month, or week.
-        If a user visits a dataset page multiple times in a day, week, month or year, it will be counted as one unique visit.
+        {{ _("This dashboard considers 'Unique visits' as the number of unique users that have visited a dataset") }}
+        {{ _("in the last year, month, or week.") }}
+        {{ _("If a user visits a dataset page multiple times in a day, week, month, or year, it will be counted as one unique visit.") }}
```

Comment:
```suggestion
        {% trans %}
        This dashboard considers 'Unique visits' as the number of unique users that have visited a dataset
        in the last year, month, or week.
        If a user visits a dataset page multiple times in a day, week, month, or year, it will be counted as one unique visit.
        {% endtrans %}
```
For big text blocks, `{% trans %}` it's better. You can look for them in CKAN core code for more examples.
Apply here and in other part of this PR.